documentation/modules/proc-upgrading-router-pods.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrading.adoc
+
+[id='post-upgrade-step-router-pod-{context}']
+= Upgrading router pods running a previous version of the image
+
+When upgrading, sometimes a router pod is not upgraded. The affected router pod can be identified and restarted to fix this issue.
```

Comment:
Perhaps "When upgrading, sometimes a router pod is not automatically restarted by {KubePlatform}.  If this happens it is important to manually restart the affected pod in order to ensure the correct operation of the system"